--- prison-1.2.0/lib/prison/CMakeLists.txt.omv~	2014-08-17 16:50:22.824993970 +0200
+++ prison-1.2.0/lib/prison/CMakeLists.txt	2014-08-17 16:54:33.998853865 +0200
@@ -27,6 +27,9 @@ SET(    prison_SRC
 
 add_library(prison SHARED ${prison_SRC})
 target_link_libraries(prison ${DMTX_LIBRARIES} ${QRENCODE_LIBRARIES} ${QT_QTCORE_LIBRARY} ${QT_QTGUI_LIBRARY})
+if(QT5_BUILD)
+	target_link_libraries(prison Qt5Gui Qt5Widgets)
+endif()
 
 set_target_properties(prison PROPERTIES VERSION "${PRISON_VERSION_STRING}" SOVERSION "${PRISON_VERSION_MAJOR}" LINK_INTERFACE_LIBRARIES "" DEFINE_SYMBOL BUILDING_PRISON)
 
